src/cli.js

```diff
@@ -3,6 +3,11 @@
 require('yargs')
     .commandDir('commands')
     .demandCommand()
+    .option('output', {
```

Comment:
Change this to be `--json` so that it's consistent with [Financial-Times/ebi](https://github.com/Financial-Times/ebi).